packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -724,6 +735,10 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
 
   protected readonly serverlessV2MinCapacity: number;
   protected readonly serverlessV2MaxCapacity: number;
+  /**
+   * The number of seconds until a serverless cluster is paused.
+   */
+  protected readonly secondsUntilAutoPause: number;
```

Comment:
```suggestion
  protected readonly secondsUntilAutoPause?: number;
```

Please refer to the following comment.​​​​​​​​

https://github.com/aws/aws-cdk/pull/32787#discussion_r1947501337